### PR TITLE
[BTSSS-79139] Claims status design updates

### PR DESCRIPTION
**Pull request by hinzed1127**
## Summary

- fetches and displays (currently only mocked) claims from the `travel_pay` API

## Related issue(s)

vets-api PR: [department-of-veterans-affairs/vets-api#16472](https://github.com/department-of-veterans-affairs/vets-api/pull/16472)
mockdata PR: [department-of-veterans-affairs/vets-api-mockdata#468](https://github.com/department-of-veterans-affairs/vets-api-mockdata/pull/468)

## Testing done

- unit tests on status page, now that designs are closer to a final form, are TBD/forthcoming

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |       |
| Desktop |        |       |

## What areas of the site does it impact?

* just travel-claims, which is not yet in prod

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [x] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

File: src/applications/travel-pay/components/TravelClaimCard.jsx [hunk 1]
```diff
@@ -14,44 +15,55 @@ const CLAIM_STATUS = {
...
     createdOn,
     claimStatus,
-    claimName,
+    // claimName,
     claimNumber,
-    appointmentName,
-    appointmentDate,
+    // appointmentName,
+    appointmentDate: appointmentDateTime,
+    appointmentLocation,
```
Comment (kjduensing) [inline src/applications/travel-pay/components/TravelClaimCard.jsx:35]: We also need the clinic name, but I forget - is that represented in `appointmentLocation` here?

**Review by kjduensing — DISMISSED** [review]
Looks good 🐠  - just a reminder to clean up any commented code.

**Comment by hinzed1127** [comment]
@kjduensing re-requesting review. Previous approval got undone after lots of cleanup/copy updates

File: src/applications/travel-pay/components/Breadcrumbs.jsx [hunk 2]
```diff
@@ -12,12 +12,12 @@ export default function BreadCrumbs() {
           label: 'VA.gov home',
         },
         {
-          href: '/health-care',
-          label: 'Health Care',
+          href: '/my-health',
+          label: 'My HealtheVet',
         },
         {
           href: '/health-care/file-travel-claim',
```
Comment (kjduensing) [inline src/applications/travel-pay/components/Breadcrumbs.jsx:19]: If the above has changed to `/my-health`, should this line change to `href: '/my-health/file-travel-claim`?

Comment (hinzed1127) [reply src/applications/travel-pay/components/Breadcrumbs.jsx:19]: Semantically, you're probably right. Given the way routing is right now with the app registry, we'd have to make any changes to this value there as well. I'd say incorporate that with all the other routing updates when we get a base URL nailed down

Comment (kjduensing) [reply src/applications/travel-pay/components/Breadcrumbs.jsx:19]: Can you add this item to a new or existing ticket? If we're not doing it now, I want to make sure we're doing it in conjunction with something else.

Comment (hinzed1127) [reply src/applications/travel-pay/components/Breadcrumbs.jsx:19]: [Created](https://app.zenhub.com/workspaces/beneficiary-travel-btsss-65147e21930cd900223d8e64/issues/gh/department-of-veterans-affairs/va.gov-team/81927)!

**Review by kjduensing — DISMISSED** [review]
Looks good! 🏆

**Review by kjduensing — APPROVED** [review]
Looks good

**Review by pjhill — APPROVED** [review]

**Pull request merged**